analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -152,7 +152,9 @@ class Maven(
             workingDir
         }
 
-        val vcsFallbackUrls = listOfNotNull(mavenProject.scm?.url, mavenProject.url)
+        val browsableScmUrl = MavenSupport.getOriginalScm(mavenProject)?.url
```

Comment:
Commit message: This is actually a fixup for 051a60b.

Reply:
Done.